azurerm/internal/services/sql/tests/resource_arm_sql_firewall_rule_test.go

```diff
@@ -218,13 +233,28 @@ resource "azurerm_resource_group" "test" {
   location = "%s"
 }
 
+resource "azurerm_storage_account" "test" {
+    name                     = "accteststore%s"
+    resource_group_name      = azurerm_resource_group.test.name
+    location                 = azurerm_resource_group.test.location
+    account_tier             = "Standard"
+    account_replication_type = "LRS"
+}
+
 resource "azurerm_sql_server" "test" {
   name                         = "acctestsqlserver%d"
   resource_group_name          = azurerm_resource_group.test.name
   location                     = azurerm_resource_group.test.location
   version                      = "12.0"
   administrator_login          = "mradministrator"
   administrator_login_password = "thisIsDog11"
+
+  extended_auditing_policy {
+    storage_endpoint                        = azurerm_storage_account.test.primary_blob_endpoint
+    storage_account_access_key              = azurerm_storage_account.test.primary_access_key
+    storage_account_access_key_is_secondary = true
+    retention_in_days                       = 6
+  }
```

Comment:
this shouldn't be needed - unfortunately this is a bug in the Azure API (being tracked in https://github.com/Azure/azure-rest-api-specs/issues/11271) which should be reverted soon, as such could we revert this for the moment?

Reply:
Sure Tom, removed and fixed it